modules/network/src/test/java/org/apache/ignite/network/DefaultMessagingServiceTest.java

```diff
@@ -168,6 +169,9 @@ private static void configureNetworkDefaults(
         lenient().when(networkConfigView.portRange()).thenReturn(0);
         lenient().when(networkConfigView.outbound()).thenReturn(outboundConfig);
         lenient().when(networkConfigView.inbound()).thenReturn(inboundConfig);
+        SslView ssl = mock(SslView.class);
+        lenient().when(ssl.enabled()).thenReturn(false);
```

Comment:
I think these mocks should be rewritten using `@InjectConfiguration`

Reply:
Do you mean It would be nice to rewrite all mocks here or just SslConfiguration?

Reply:
all mocks here (I don't expect it to be a big change)